system/internal/service/user.go

```diff
@@ -121,6 +158,7 @@ func (svc *user) FindOrCreate(user *types.User) (out *types.User, err error) {
 		out, err = svc.user.FindBySatosaID(user.SatosaID)
 
 		if err == repository.ErrUserNotFound {
+			// @todo do we allow autocreation of nonexisting users?
```

Comment:
This is weird, no.

Reply:
This (logic) is still needed but ONLY in context of external logins, where we create users on-the-fly on successful (external) authentication. Auth service already does that.

Removed from user service, added comment to auth service.